packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7071,3 +7071,52 @@ class _StatefulBuilderState extends State<StatefulBuilder> {
   @override
   Widget build(BuildContext context) => widget.builder(context, setState);
 }
+
+/// A widget that paints its area with a specified [Color] and then draws its
+/// child.
+class ColoredBox extends SingleChildRenderObjectWidget {
+  /// Creates a widget that paints its area with the specified [Color].
+  ///
+  /// The `color` parameter must not be null.
+  const ColoredBox({@required this.color, Widget child, Key key})
+      : assert(color != null),
+        super(key: key, child: child);
+
+  /// The color to paint the background area with.
+  final Color color;
+
+  @override
+  RenderColoredBox createRenderObject(BuildContext context) {
+    return RenderColoredBox(color: color);
+  }
+}
+
+/// A render object that paints its area with the specified [Color].
+class RenderColoredBox extends RenderProxyBox {
```

Comment:
Does this class need to be public?

Reply:
I suppose not.  Making it private and will update the test.